### DISCUSSION TITLE
May I ask how to solve the Chinese garbled characters of the web site

### DISCUSSION THREAD

**Discussion by hhly** [CoderLine/alphaTab#986] [Q&A] [answered]
May I ask how to solve the Chinese garbled characters of the web site
<img width="755" alt="image" src="https://user-images.githubusercontent.com/29226405/191899074-4f0f23b4-841e-4e84-a7a3-310a8dc148ee.png">

**Comment by Danielku15** [comment, marked as answer]
There have been many discussions around the handling of chinese character sets. I recommend you to search through the plenty of discussions and issues around this topic.

**Reply by hhly** [reply]
Excuse me, how to set the capo in alphatab, I can't find it in the document

**Reply by goodgame365** [reply]
变调夹是在GTP里设置的吧，如果想在alphatab修改显示位置和相关，可以在 这个源文件里修改
src\rendering\effects\CapoEffectInfo.ts 这里可以修改
